lib/skate_web/router.ex

```diff
@@ -70,6 +70,7 @@ defmodule SkateWeb.Router do
     get "/minischedule/run/:trip_id", MinischeduleController, :run
     get "/minischedule/block/:trip_id", MinischeduleController, :block
     get "/intersection", IntersectionController, :intersection
+    get "/vehicle_for_trips", VehicleForTripController, :show
```

Comment:
If we want the VPP to either get updates if the vehicle on those trips changes, or get realtime updates if the bus is on a non-selected route, this would need to be a channel instead of a GET.

Reply:
I haven't heard any talk of updating if the vehicle on trip changes (nor against it), but yeah presumably we do want realtime updates. Will rework.